test/membrane_mp4/demuxer/isom/integration_test.exs

```diff
@@ -56,6 +56,39 @@ defmodule Membrane.MP4.Demuxer.ISOM.IntegrationTest do
     perform_test(pipeline, in_path, out_path)
   end
 
+  @tag :tmp_dir
+  test "single aac track payloaded and depayloaded", %{tmp_dir: dir} do
+    in_path = "test/fixtures/in_audio.aac"
+    out_path = Path.join(dir, "out")
+
+    structure =
+      child(:file, %Membrane.File.Source{location: in_path})
+      |> child({:parser, :in}, %Membrane.AAC.Parser{
+        in_encapsulation: :ADTS,
+        out_encapsulation: :none
+      })
+      |> child(:payloader, Membrane.MP4.Payloader.AAC)
+      |> child(:depayloader, Membrane.MP4.Depayloader.AAC)
+      |> child({:parser, :out}, %Membrane.AAC.Parser{
```

Comment:
don't we want to do the same trick as with the h264 payloder - that is, prepare a bin, that would have a parser?

Reply:
In AAC.Payloader, the buffers remain unchanged, so there is no need to parse them afterward.